lib/pbkit/pbkit.c

```diff
@@ -2260,8 +2260,10 @@ DWORD *pb_end(DWORD *pEnd)
 #endif
 
 #ifdef DBG
+    if (pEnd!=pb_PushNext) debugPrint("pb_end: input pointer invalid o r not following previous write addresses\n");
     if (pb_BeginEndPair==0) debugPrint("pb_end without a pb_start\n");
     pb_BeginEndPair=0;
+    pb_PushNext=NULL;
```

Comment:
nit: actually we don't need this `NULL`-ing, because `pb_BeginEndPair` probably guards all use? Otherwise, for completeness, we'd probably also want to `NULL` the other globals (I believe `pb_PushIndex`?)

Reply:
I feel like we'd rather have less NULLings than more - removed the `pb_PushNext=NULL;` line.